sqlalchemy_filters/filters.py

```diff
@@ -1,6 +1,11 @@
 # -*- coding: utf-8 -*-
 from collections import Iterable, namedtuple
-from inspect import signature
+try:
+    from inspect import signature
+except ImportError:
+    # For python2 capability. NOTE: you must fulfill this requirement
+    # yourself, it is not in requirements.txt
+    from funcsigs import signature
```

Comment:
This would be great to cover by tests in [TOX](https://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix) or travis matrix with `funcsigs` explicitly deployed for a python2 environment.

Reply:
I agree. If we're adding this new logic, it'd be great to cover it with tests.